hoomd/hpmc/pytest/test_shape_move.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+from collections.abc import Sequence, Mapping
+import math
+from numbers import Number
+import hoomd
+import hoomd.write
+import hoomd.hpmc
+import numpy as np
+import pytest
+from copy import deepcopy
+import coxeter
+from coxeter.shapes import ConvexPolyhedron
+from hoomd.logging import LoggerCategories
+from hoomd.conftest import logging_check
+
+_seed = 0
+
+
+def _equivalent_data_structures(reference, struct_2):
+    """Compare arbitrary data structures for equality.
+
+    ``reference`` is expected to be the reference data structure. Cannot handle
+    set like data structures.
+    """
+    if isinstance(reference, np.ndarray):
+        return np.allclose(reference, struct_2)
+    if isinstance(reference, Mapping):
+        # if the non-reference value does not have all the keys
+        # we don't check for the exact same keys, since some values may have
+        # defaults.
+        if set(reference.keys()) - set(struct_2.keys()):
+            return False
+        return all(
+            _equivalent_data_structures(reference[key], struct_2[key])
+            for key in reference)
+    if isinstance(reference, Sequence):
+        if len(reference) != len(struct_2):
+            return False
+        return all(
+            _equivalent_data_structures(value_1, value_2)
+            for value_1, value_2 in zip(reference, struct_2))
+    if isinstance(reference, Number):
+        return math.isclose(reference, struct_2, rel_tol=1e-4)
+
+
+ttf = coxeter.families.TruncatedTetrahedronFamily()
```

Comment:
Remove the coxeter dependency. This appears to be a basic unit test. You could use a simpler, hand-coded test shape (e.g. a bipyramid with one moving vertex) to ensure that the API functions.